bin/cron/admin_progress

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+#
+# This script caches database queries whose results are displayed at studio.code.org/admin/progress.
+#
+require_relative('../../dashboard/config/environment')
+require 'cdo/properties'
+
+def main()
+  return unless only_one_running?(__FILE__)
+
+  SeamlessDatabasePool.use_persistent_read_connection do
+    user_count = User.count
+
+    levels_all = User.joins(:user_levels).group(:level_id)
+    levels_attempted = levels_all.where('best_result > 0').count
```

Comment:
Why is this being copied from the admin_reports controller, rather than creating some kind of helper for use in both these places?